lib/xcsoar/mapgen/server/server.py

```diff
@@ -12,6 +12,14 @@
 from xcsoar.mapgen.georect import GeoRect
 from xcsoar.mapgen.waypoints.parser import parse_waypoint_file
 
+cherrypy.config.update(
+    {
+        "log.screen": True,  # Log to stdout
+        "log.error_file": "error.log",  # Log errors to a file
+        "log.access_file": "access.log",  # Log access to a file
+    }
+)
+
```

Comment:
Normally this runs in a container. The Filesystem is usually readonly. Please remove this configuration.